source/myukfast/accounts.md

```diff
@@ -0,0 +1,27 @@
+# Accounts
+
+```eval_rst
+   .. title:: MyUKFast | Accounts
+   .. meta::
```

Comment:
Same here. as accounts could mean the moneytary side of accounts or a myukfast account, or user account